opennms-services/src/main/java/org/opennms/netmgt/enlinkd/NodeDiscoveryBridge.java

```diff
@@ -142,79 +141,74 @@ protected void runCollection() {
     }
 
     private BridgeElement getDot1dBridgeBase(SnmpAgentConfig peer) {
-        String trackerName = "dot1dbase";
-        final Dot1dBaseTracker dot1dbase = new Dot1dBaseTracker();
-        SnmpWalker walker = SnmpUtils.createWalker(peer, trackerName,
-                                                   dot1dbase);
-        walker.start();
-
         try {
-            walker.waitFor();
-            if (walker.timedOut()) {
-                LOG.info("run:Aborting Bridge Linkd node scan : Agent timed out while scanning the {} table",
-                         trackerName);
+            final Dot1dBaseTracker dot1dbase = new Dot1dBaseTracker();
+            
+            try {
+                m_linkd.getLocationAwareSnmpClient().walk(getPeer(), dot1dbase).
+                withDescription("dot1dbase").
+                atLocation(getLocation()).
+                execute().
+                get();
+           } catch (ExecutionException e) {
+                    // pass
```

Comment:
We should log the exception, and treat it similar to the timeout behavior we had previously.